var/spack/repos/builtin/packages/py-minis-validation/package.py

```diff
@@ -25,7 +26,8 @@ class PyMinisValidation(PythonPackage):
     depends_on('py-pyyaml@5.1:', type='run')
     depends_on('py-dask-mpi@2.0:', type='run')
     depends_on('neuron+python@7.8:', type='run')
-    depends_on('py-bluepy@0.14:', type='run')
+    depends_on('py-bluepy@0.14:0.99', type='run')
+    depends_on('py-neurom@1.4:1.59', type='run')  #temp fix for h5py < 3
```

Comment:
curious about this comment here : can you briefly explain the change in dependency with h5py < 3?

Reply:
h5py 3 brings breaking changes. The latest bluepy@0.16.0 can't use it yet because of that. Neurom 1.6 already updated its h5py dependency to be > 3. That's why this fixing of versions.

Reply:
Ok.
flake8 is complaining for comment (space is needed after #)

Reply:
Thank you! but I want to experiment here for now, so it is not ready for review/merging.

Reply:
Ah ok! thanks for the note otherwise I would have pressed merge button :)

Reply:
No no, no merge button please :) There are some silent problems with bglibpy + bluepy. I'm surprised no one complained about them. The latest deployed bglibpy requires bluepy >= 0.16, but the latest deployed version of bluepy is 0.14.15. We need to have more thorough tests for modules deployment.

Reply:
Ok! I converted this to Draft ! 

Note that Spack CI and BB5 deployment CI will be turned off from tomorrow until 2nd of week of Jan for new deployment (as announced earlier). So if anything needs to be merged for the month of December then it's today. :)

Reply:
thank you, I will do my best to finish it today